pkg/agent/datamodel/types.go

```diff
@@ -676,6 +676,7 @@ type KubernetesConfig struct {
 	MaximumLoadBalancerRuleCount      int               `json:"maximumLoadBalancerRuleCount,omitempty"`
 	PrivateAzureRegistryServer        string            `json:"privateAzureRegistryServer,omitempty"`
 	NetworkPluginMode                 string            `json:"networkPluginMode,omitempty"`
+	EbpfDataplane                     int               `json:"ebpfDataplane,omitempty"`
```

Comment:
should this be a bool instead?

maybe an enum of you can have other values than 1?

Reply:
The struct is meant to match the AKS struct here:
https://msazure.visualstudio.com/CloudNativeCompute/_git/aks-rp?path=/protos/hcp/types/common/v1/kubernetesconfig.pb.go&version=GBmaster&line=216&lineEnd=217&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents

Updated to a matching enum.

Reply:
thanks, please also refrain from sharing ADO links within this repo in the future